x-pack/plugins/security_solution/server/endpoint/routes/actions/utils.ts

```diff
@@ -70,6 +70,11 @@ const COMMANDS_WITH_ACCESS_TO_FILES: CommandsWithFileAccess = deepFreeze<Command
     sentinel_one: false,
     crowdstrike: false,
   },
+  runscript: {
+    endpoint: false,
+    sentinel_one: false,
+    crowdstrike: true,
```

Comment:
Just to confirm - `runscript` for crowdstrike will have a file available for the user to download?

Reply:
Good question 👍  Answer: no - in CrowdStrike `runscript` always returns `stdout` and no file. However, a script can be designed to generate a file on the host. Then to retrieve this file - there is another command `get` that will have to be implemented.

Reply:
Got it. so this setting here should be `false` for `crowdstrike` since the command does not return a file for download

Reply:
Oh, good catch, I didn't notice it's `COMMANDS_WITH_ACCESS_TO_FILES` - it looked like the other `manual/automated` map. Thanks!

Reply:
Q: Looks like this is yet to be updated to `false`?